### PR TITLE
Initial support for takes packaging

### PR DESCRIPTION
**Pull request by amagdas**
Please ignore all the logging, it's going to be removed before creating the PR to master.

### CODE REVIEW

**Comment by mdhaman** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

File: server/apps/archive/archive_composite.py [hunk 1]
```diff
@@ -61,6 +63,53 @@ def get_item_ref(item):
 
 class PackageService():
```
Comment (sivakuna-aap) [inline server/apps/archive/archive_composite.py]: This should extend from `BaseService`

Comment (amagdas) [inline server/apps/archive/archive_composite.py]: Nop, it's not a resource service, more like a helper service. You can't get it through `get_resource_service`

**Pull request merged**